README.md

```diff
@@ -0,0 +1,23 @@
+# Whistler ###
+
+#### Whistler is an error-logging service that generates tickets and allows end-users to report additional details.
+
+
+##### Docker #####
+```
+$ docker build server/
```

Comment:
Yes. There's a .env-template file to base it off of. They'll need to register for a google developer app account as well to get an appId/secret.